audio/midi/fluid/voice.cpp

```diff
@@ -1816,7 +1816,7 @@ void Sample::optimize()
             s->loopstart = s->start;
             }
 
-      IF_ASSERT_FAILED(s->loopend <= s->end) {
+      if (s->loopend <= s->end) {
```

Comment:
This is not an equal replacement

Condition
IF_ASSERT_FAILED(s->loopend <= s->end)

Equals to
if (!(s->loopend <= s->end))

Or just
if (s->loopend > s->end)

Reply:
oops, thanks for catching